common/src/main/java/com/wynntils/features/user/BeaconBeamFeature.java

```diff
@@ -47,35 +47,45 @@ public void onTrackedQuestUpdate(TrackedQuestUpdateEvent event) {
     }
 
     @SubscribeEvent
-    public void onRenderLevelLast(RenderLevelLastEvent event) {
+    public void onRenderLevelLast(RenderTileLevelLastEvent event) {
         if (CompassModel.getCompassLocation().isEmpty()) return;
 
         PoseStack poseStack = event.getPoseStack();
-        MultiBufferSource.BufferSource bufferSource =
-                MultiBufferSource.immediate(Tesselator.getInstance().getBuilder());
+        MultiBufferSource.BufferSource bufferSource = MultiBufferSource.immediate(new BufferBuilder(256));
```

Comment:
Where did 256 come from?

Reply:
I don't remember

Reply:
Does this even have to be changed?

Reply:
Perhaps use `MultiBufferSource.BufferSource source = McUtils.mc().renderBuffers().bufferSource();`?

Reply:
Yeah, that is what is used internally